designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ManagementResourcesSpec.scala

```diff
@@ -178,17 +176,6 @@ class ManagementResourcesSpec
     }
   }
 
-  test("rejects deploy without comment if comment required") {
```

Comment:
We still need this test, validation should work exactly as before the change.

Reply:
But why do we want this in ManagementResourcesSpec? This should only be responsible for routing and the validation should be done in DeploymentService now?

Reply:
Good point, I havent noticed that it was moved to DeploymentServiceSpec (which is responsible for executiong validations on action parameters).